go/pserver/service.go

```diff
@@ -14,6 +24,10 @@ const (
 	Uninitialized      = "pserver not fully initialized"
 )
 
+const (
+	checkpoint_path = "./checkpoints/"
```

Comment:
Go's naming convention is camelCase, not snake_case.

checkpointPath need to be an argument (`flag.String`) passed to go/cmd/pserver program. Since the k8s will configure the path.